### PR TITLE
networkctl: add page and Polish translation

### PR DESCRIPTION
**Pull request by acuteenvy**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/networkctl.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# networkctl
+
+> Query the status of network links.
+> Manage the network configuration using systemd-networkd.
```
Comment (kbdharun) [inline pages/linux/networkctl.md:4]:
```suggestion
> Manage the network configuration using `systemd-networkd`.
```

File: pages.pl/linux/networkctl.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# networkctl
+
+> Zapytaj o stan łączy sieciowych.
+> Zarządzaj konfiguracją sieci za pomocą systemd-networkd.
```
Comment (kbdharun) [inline pages.pl/linux/networkctl.md:4]:
```suggestion
> Zarządzaj konfiguracją sieci za pomocą `systemd-networkd`.
```

**Review by kbdharun — COMMENTED** [review]
A minor suggestion and the PR is GTG

**Comment by acuteenvy** [comment]
I applied your suggestion, but now that I think of it, systemd-networkd is not a command that should be run directly, it's a daemon started automatically by systemd. And according to [this section of the style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases) only commands should be surrounded with backticks. I'll leave it as is though if you think this is better.

**Comment by acuteenvy** [comment]
So should I remove it or leave it as it is?

**Comment by kbdharun** [comment]
> So should I remove it or leave it as it is?

I guess it is fine

**Review by blueskyson — APPROVED** [review]
It's fine because no real-world environment labels an interface as "interface1".

**Pull request merged**